lib/learn_test/strategies/rspec.rb

```diff
@@ -25,6 +25,10 @@ def configure
           argv << '--fail-fast'
         end
 
+        if example_option_present?
+          argv << "--example #{options[:example]}"
```

Comment:
```ruby
argv << "#{options[:example].collect{|e| "--example #{e}"}.join(" ")}"
```

Reply:
The only thing I don't like about this is it assumes `options[:example]` is an array but that's okay as we set it earlier upon detection of the `--example` flag. We could be more defensive but I'd be okay with that.

Reply:
We could check the class type of `options[:example]` here and perform different actions?

Even `[options[:example]].flatten` would work but that's ugly